app/controllers/contest_registrations_controller.rb

```diff
@@ -248,10 +248,12 @@ def destroy
   def index
     registrations = @contest.contest_registrations.includes(:user)
```

Comment:
I think we can do 
```ruby
registrations = @contest.contest_registrations.includes(:user, :team)
```
to avoid additional SQL query?

Reply:
Sure.